lists.rb

```diff
@@ -0,0 +1,69 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'colors_utils'
+
+# rubocop: disable Layout/LineLength
+# rubocop: disable Style
```

Comment:
- [ ] Linting is important to reduce errors and improve the overall quality of your code  so I think fixing them would be much better than disabling them

Reply:
Resolved.

Reply:
This issue is well resolved! ✔️